src/5e-SRD-Magic-Items.json

```diff
@@ -2138,419 +4485,973 @@
+      "Potion, very rare",
+      "When you drink this potion, your Strength score changes to 27 for 1 hour. The potion has no effect on you if your Strength is equal to or greater than that score.",
+      "This potion's transparent liquid has floating in it a sliver of fingernail from a cloud giant."
     ],
-    "url": "/api/magic-items/periapt-of-health"
+    "url": "/api/magic-items/potion-of-giant-strength-cloud"
   },
   {
-    "index": "periapt-of-proof-against-poison",
-    "name": "Periapt of Proof against Poison",
+    "index": "potion-of-giant-strength-storm",
+    "name": "Potion of Storm Giant Strength",
     "equipment_category": {
-      "index": "wondrous-items",
-      "name": "Wondrous Items",
-      "url": "/api/equipment-categories/wondrous-items"
+      "index": "potion",
+      "name": "Potion",
+      "url": "/api/equipment-categories/potion"
+    },
+    "rarity": {
+      "name": "Legendary"
     },
+    "variants": [],
     "desc": [
-      "Wondrous item, rare",
-      "This delicate silver chain has a brilliant-cut black gem pendant. While you wear it, poisons have no effect on you. You are immune to the poisoned condition and have immunity to poison damage."
+      "Potion, legendary",
+      "When you drink this potion, your Strength score changes to 29 for 1 hour. The potion has no effect on you if your Strength is equal to or greater than that score.",
+      "This potion's transparent liquid has floating in it a sliver of fingernail from a storm giant."
     ],
-    "url": "/api/magic-items/periapt-of-proof-against-poison"
+    "url": "/api/magic-items/potion-of-giant-strength-storm"
   },
   {
-    "index": "periapt-of-wound-closure",
-    "name": "Periapt of Wound Closure",
+    "index": "potion-of-growth",
+    "name": "Potion of Growth",
     "equipment_category": {
-      "index": "wondrous-items",
-      "name": "Wondrous Items",
-      "url": "/api/equipment-categories/wondrous-items"
+      "index": "potion",
+      "name": "Potion",
+      "url": "/api/equipment-categories/potion"
+    },
+    "rarity": {
+      "name": "Uncommon"
     },
+    "variants": [],
     "desc": [
-      "Wondrous item, uncommon (requires attunement)",
-      "While you wear this pendant, you stabilize whenever you are dying at the start of your turn. In addition, whenever you roll a Hit Die to regain hit points, double the number of hit points it restores."
+      "Potion, uncommon",
+      "When you drink this potion, you gain the \"enlarge\" effect of the enlarge/reduce spell for 1d4 hours (no concentration required). The red in the potion's liquid continuously expands from a tiny bead to color the clear liquid around it and then contracts. Shaking the bottle fails to interrupt this process."
     ],
-    "url": "/api/magic-items/periapt-of-wound-closure"
+    "url": "/api/magic-items/potion-of-growth"
   },
   {
-    "index": "philter-of-love",
-    "name": "Philter of Love",
+    "index": "potion-of-healing",
```

Comment:
How do we want to handle this item? This is a case where the variant would have the same name as the parent item

Reply:
For now, I added an item with the same name, but with an id of `potion-of-healing-common`

Reply:
I love how the SRD always has a weird curve ball for everything.

I think that's reasonable for now and we can adjust later.